src/pl/plr.spec

```diff
@@ -3,10 +3,14 @@ Name:           plr-hawq
 Version:        08.03.00.14
 Release:		0
 Prefix:         /usr/local
-License:		GPL
+License:		Apache
+BuildRequires:  R-devel >= 3.1.0
```

Comment:
Now HAWQ have many dependencies, 'thrift', 'boost', 'curl', etc. Each have the concerns you mentioned. 

If this really a concern for plr, then I advise to test with the latest one on both centos6 and centos7.